pyproject.toml

```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
 name = "nautobot-secrets-providers"
-version = "1.4.1-beta.1"
+version = "1.4.1b1"
```

Comment:
We follow the `1.4.1-beta.1` convention in Core. Is there a reason to change this?

I know there is a test failing but I think that test can be removed/changed.

Reply:
We should IMHO use normalized version in `pyproject.toml` to have Docker image tags on par with pip package versions.

Reply:
Interesting... Reading Poetry documentation, it's version command, PEP440 (https://peps.python.org/pep-0440/#semantic-versioning) it should have always been this way. Curious @nautobot/maintain-core, any prior knowledge why we use the full SemVer in Core?

Reply:
Seems like this change/clarification happen last year: https://github.com/python-poetry/poetry/pull/5576